src/components/UI/Buttons/Buttons.module.sass

```
@@ -25,3 +26,15 @@ $button-border: $color-primary
 
     &:hover
         background-color: $color-primary
+
+.button-icon
+    width: 20px
+    height: 20px
+
```

- Shouldn't be a class, but `button > img` selector.
- Shouldn't use `px` units, use `em` instead.
```suggestion

button > img
    width: 1.5em
    aspect-ratio: 1

```